lab01/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Incremental Game</title>
+        <h1>Incremental Game Framework</h1>
+        <link rel="stylesheet" type="text/css" href="app.css">
+    </head>
+    
+    <body>
+        
+        <div class="lines">
+            Stroy Line 1<br/>
+            Stroy Line 2<br/>
+            Stroy Line 3<br/>
+            Stroy Line 4<br/>
+        </div>
+        
+        <div class="resources">
+            <em>Resource Name:</em>
+            <em>15568</em>
+            <em>
+                <button>Incrementer</button>
+            </em>
```

Comment:
not sure why you would need <em> to surround button element.

Reply:
em should be used only to emphasize some text not to be used on every single text.